client/question/components/LearnerViewDirective.js

```diff
@@ -896,10 +897,14 @@ tie.directive('learnerView', [function() {
         $scope.setEventListenerForVisibilityChange();
 
         /**
-         * Shows the privacy modal on click.
+         * Shows the privacy modal on click if a URL is not specified.
```

Comment:
"Shows the privacy modal on click if a URL to a separate Privacy Page does not exist."

Reply:
Done.